### DISCUSSION TITLE
Custom model name?

### DISCUSSION THREAD

**Discussion by pboling** [janko/rodauth-model#3] [General]
Is it possible to use an alternative to `Account`, or perhaps a namespaced Account model, like `MyApp::Account`?

**Comment by janko** [comment]
Yes, if you're using rodauth-rails, it will automatically infer the model class from the `accounts_table` value. And if you need a namespaced account model, there is the `rails_account_model` setting:

```rb
rails_account_model { MyApp::Account }
```

**Reply by pboling** [reply]
I have a situation where the table name is default with `accounts`, but I need to namespace the model, so it wouldn't be deriveable from the table name.  Will try out `rails_account_model`, thanks!

**Reply by pboling** [reply]
Also, it might be useful to those like me to have that be one of the commented options in the default generated file near the line:
```
    # Change prefix of table and foreign key column names from default "account"
    # ... existing examples & defaults
    # rails_account_model { MyApp::Account }
```
